lib/memory.c

```diff
@@ -127,7 +127,6 @@ void *m0_alloc(size_t size)
 {
 	void *area;
 
-	M0_ENTRY("size=%zi", size);
```

Comment:
`Avoid redundant logs`?
What are the redundant logs beside these two lines?

Sometimes, the two lines of debug message are helpful.

Reply:
@huanghua78   malloc() entry and exit logs are occupying highest percentage of m0trace log, among other msgs. And  as per  discussion with @nikitadanilov , we though of removing them as they are not much useful for debugging.
@nikitadanilov  please provide your feedback